### DISCUSSION TITLE
Callback method for filter onchange

### DISCUSSION THREAD

**Discussion by acejusto11** [TanStack/table#4415] [Q&A]
Hi,

Is there a callback function for on filter changed in react-table v7? I am looking for a method in the old version of react-table where I can pass a callback function everytime I changed the filter in the table.

Thank you

**Comment by joelbong** [comment]
This feature would be much appreciated. I'm on version 8.10.3. An event on table like, onFiltered: (newData) => void (indepedent if its globally or from a column). 
If there is any other way i can implement this, let me know. I tried several approach, like patching getFilteredRowModelPatch, overriding onColumnFiltersChange and onGlobalFilterChange, but these didn't bring me desired result.
